fabric-rendering-v1/src/client/java/net/fabricmc/fabric/mixin/client/rendering/GameRendererMixin.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.client.rendering;
+
+import com.llamalad7.mixinextras.sugar.Local;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.gui.render.GuiRenderer;
+import net.minecraft.client.render.BufferBuilderStorage;
+import net.minecraft.client.render.GameRenderer;
+import net.minecraft.client.render.VertexConsumerProvider;
+import net.minecraft.client.render.item.HeldItemRenderer;
+
+import net.fabricmc.fabric.impl.client.rendering.SpecialGuiElementRegistryImpl;
+
+@Mixin(GameRenderer.class)
+public class GameRendererMixin {
+	@Shadow
+	@Final
+	private GuiRenderer guiRenderer;
+
+	@Inject(method = "<init>", at = @At(value = "RETURN"))
+	private void initializeSpecialElements(MinecraftClient client, HeldItemRenderer firstPersonHeldItemRenderer, BufferBuilderStorage buffers, CallbackInfo ci, @Local VertexConsumerProvider.Immediate immediate) {
+		GuiRendererAccessor accessor = (GuiRendererAccessor) this.guiRenderer;
+		SpecialGuiElementRegistryImpl.onReady(client, immediate, accessor.getSpecialElementRenderers());
+	}
```

Comment:
I don't think we need this mixin if we have the `GuiRenderer` mixin. The only thing we get from this mixin is the client instance, which can be easily retrieved.